### PR TITLE
Add bicyle and motorcycle parking merge for Bordeaux Metropole

### LINKED ISSUE #1170: Add bicyle parking merge for Bordeaux Metropole
Remove for Bordeaux city, need to be replace with data from Bordeaux Metropole
https://opendata.bordeaux-metropole.fr/explore/dataset/st_arceau_p/information/

### PR DESCRIPTION
**Pull request by gileri**
Fixes #1170

### CODE REVIEW

File: analysers/analyser_merge_cycle_parking_FR_bm.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                    types=["nodes", "ways"],
+                    tags={"amenity": "bicycle_parking"}),
+                conflationDistance=20,
+                mapping=Mapping(
+                    static1={"amenity": "bicycle_parking"},
+                    static2={"source": self.source},
+                    mapping1={
+                        "capacity": lambda res: None if res["nombre"] in (None, "0") else str(int(res["nombre"])*2),
+                        "cargo_bike": lambda res: "yes" if res["typologie"] == "ARCEAU_VELO_CARGO" else None,
+                        "ref": "gid",
```
Comment (frodrigo) [inline analysers/analyser_merge_cycle_parking_FR_bm.py:58]: `gid` look like internal id. Are you sure there are stable?

Comment (gileri) [reply analysers/analyser_merge_cycle_parking_FR_bm.py]: You're right, let's be conservative and not use it for now.

**Comment by frodrigo** [comment]
Good for me. Thank for working on this.

**Comment by gileri** [comment]
No biggie, it was quite simple thankfully.

**Pull request merged**